lib/edit-site-page.php

```diff
@@ -50,7 +50,7 @@ function gutenberg_get_editor_styles() {
 	);
 
 	/* translators: Use this to specify the CSS font family for the default font. */
-	$locale_font_family = esc_html_x( 'Noto Serif', 'CSS Font Family for Editor Font', 'gutenberg' );
+	$locale_font_family = esc_html_x( '-apple-system, BlinkMacSystemFont,"Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,"Helvetica Neue", sans-serif', 'CSS Font Family for Editor Font', 'gutenberg' );
```

Comment:
If we switch to system fonts this doesn't need to be translatable since they already work for all locales out of the box. I think we can safely remove the translation function and just add the CSS directly :+1: